src/smolagents/tools.py

```diff
@@ -235,10 +235,10 @@ def save(self, output_dir):
             from typing import Optional
 
             class {class_name}(Tool):
-                name = "{self.name}"
-                description = "{self.description}"
+                name = {json.dumps(self.name)}
+                description = {json.dumps(textwrap.dedent(self.description).strip())}
```

Comment:
What about using just a multiline string?
```suggestion
                description = """{self.description}"""
```